packages/kbn-grid-layout/grid/grid_panel.tsx

```diff
@@ -60,66 +61,103 @@ export const GridPanel = forwardRef<
           gridLayoutStateManager.activePanel$,
           gridLayoutStateManager.gridLayout$,
           gridLayoutStateManager.runtimeSettings$,
+          gridLayoutStateManager.expandedPanelId$,
+          gridLayoutStateManager.isMobileView$,
```

Comment:
I think styling for the expanded panel ID + mobile view can be part of a **separate** subscription rather than the drag/drop/resize subscription - what do you think?

Reply:
> We can do it, but unfortunately everytime we expand the panel, the drag/drop/resize subscription runs anyway.

That's fine! It's more the inverse I was worried about - should we really be checking for mobile view + expanded panel as the panel is getting dragged around? This event fires **a lot** on a drag/resize event, so we should keep this subscription to a bare minimum IMO :)

Reply:
That makes sense for me, thanks for explanation!